Dockerfile

```diff
@@ -7,26 +7,21 @@ EXPOSE 8000
 
 WORKDIR /app
 
-RUN addgroup -g 10001 app && \
-    adduser -D -u 10001 -G app -h /app -s /sbin/nologin app
+RUN adduser --uid 10001 --disabled-password --gecos '' --no-create-home app
 
-RUN apk --no-cache --virtual add \
-    build-base \
-    bash \
-    curl \
-    git \
-    libffi-dev \
-    py-cffi \
-    postgresql \
-    postgresql-dev \
-    postgresql-client
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get -y install libpq-dev && \
+    apt-get -y install libffi-dev && \
+    apt-get -y install python-dev && \
+    apt-get -y install build-essential
```

Comment:
Any reason why those packages are installed in individual apt-get install calls?